ckan/nginx_ckan

```diff
@@ -16,6 +16,7 @@ http {
             alias /var/www/datajson/$1-data.json;
         }
         location / {
+            add_header X-Frame-Options SAMEORIGIN always;
```

Comment:
Can you add a comment here, something like "Avoid CSRF" or something? See similar comment on line 31: `HSTS compliance`